projects/frontend/data-pipelines/gui/e2e/fixtures/lib/manage/e2e-cypress-dp-test.json

```diff
@@ -0,0 +1,19 @@
+{
+  "job_name": "e2e-cp-$env-placeholder$-test-job",
+  "description": "Testing job, that is used in data-pipelines CI/CD for UI e2e tests. It is intended to be always available in order to speed up deployment and if deleted will be automatically re-created by the e2e tests." ,
+  "config": {
+    "db_default_type": "IMPALA" ,
+    "contacts": {
+      "notified_on_job_failure_user_error": [] ,
+      "notified_on_job_failure_platform_error": [] ,
+      "notified_on_job_success": [] ,
+      "notified_on_job_deploy": []
+    } ,
+    "schedule": {
+      "schedule_cron": "11 23 5 8 1"
```

Comment:
It would be nice if there was a comment for each cron expression explaining what it means.

Reply:
Could you provide a suggestion to understand the context?

For example, on one hand, the Frontend does include a data job details page, where an explanation is given. For example,
```
Cron expression
11 * * * * 

Next 5 executions
Feb 20, 2023, 05:11 PM UTC
Feb 20, 2023, 06:11 PM UTC
Feb 20, 2023, 07:11 PM UTC
Feb 20, 2023, 08:11 PM UTC
Feb 20, 2023, 09:11 PM UTC
```

Usually, a person uses a CLI now, to create a data job (by template). That is populated using:
https://github.com/vmware/versatile-data-kit/blob/main/projects/vdk-control-cli/src/vdk/internal/control/job/sample_job/config.ini
, and it does provide what a schedule_cron is.

There are numerous tools for helping with cronjob creation or evaluation, I sometimes use https://crontab.guru.

Or are you thinking about providing a human-readable explanation (ini file comment) for the cron expression used by this test? Because, I am not sure it is actually needed by the test itself, but ceremonies only.

Reply:
I'm just saying it would make the config more readable if we had something like

```
"schedule": {
    // At 23:11 on the 5th of every month and every Monday of August
    "schedule_cron": "11 23 5 8 1"
    ...
    }
```

Reply:
I'll consider to add a code comment, with the e2e tests change  when we will actually run these tests;
Not sure what will happen in case of discrepancy, will then the explanation be wrong, or the cron expression itself?
Not sure if the schedule_cron is actually meaningful for the test at this point, or could be any valid one, so resolving the comment